### PR TITLE
option for entity delete to wait till ops submitted

### PR DESCRIPTION
**Pull request by yak32**
deleteEntities now supports waitSubmitted option to wait till all scene ops submitted.

### CODE REVIEW

File: src/entities/delete.js [hunk 1]
```diff
@@ -176,6 +177,18 @@ async function deleteEntities(entities, options = {}) {
             }
         });
     }
+
+    if (options.waitSubmitted) {
+
+        // wait for scene ops to finish
+        await new Promise((resolve) => {
+            if (api.realtime.scenes.current) {
+                api.realtime.scenes.current.whenNothingPending(resolve);
```
Comment (yaustar) [inline src/entities/delete.js:186]: Is this current API? `whenNothingPending` is a bit of a mouthful 😅 What does this do?

Comment (slimbuck) [reply src/entities/delete.js:186]: https://share.github.io/sharedb/api/doc#whennothingpending

**Review by slimbuck — APPROVED** [review]

File: src/entities/delete.js (hunk 1 above)
Comment (yaustar) [reply src/entities/delete.js:186]: Oh, that's SharedDB? That is a mouthful and a half

Comment (slimbuck) [reply src/entities/delete.js:186]: https://share.github.io/sharedb/api/doc#whennothingpending

Comment (yak32) [reply src/entities/delete.js:188]: added a better (hopefully) comment

**Pull request merged**